seisidd/devices/beamline.py

```diff
@@ -59,8 +292,9 @@ class Counter:
 class FastShutter:
     """Fast shutter for taking dark field images"""
     # NOTE:
-    #  apstoos provides shutter class for the main/slow beamline shutter,
+    #  apstools provides shutter class for the main/slow beamline shutter,
```

Comment:
If similar shutters are also used at other beamlines, it would be the best for @prjemian to provide support instead of individual beamilnes.

Reply:
Sure, I will check to see if the FS are a common device across the beamlines.